.github/workflows/operator-ci.yml

```diff
@@ -48,12 +66,10 @@ jobs:
           github token: ${{ secrets.GITHUB_TOKEN }}
           driver: docker
           start args: '--addons=ingress'
-
       - name: Build the Keycloak Docker image
```

Comment:
I noticed that we're basically building KC image three times now. Maybe we could build it and push it to Quay right away in the first build step. WDYT?

Reply:
it's an option, but building the image is actually pretty fast (~40 seconds) and in this way we do not depend on network on 2/3 of the jobs ...
is a tradeoff, no strong opinion here.

Reply:
We're uploading the image in any case for the OLM tests, so IMO we can benefit from that in other tests too.

Reply:
done

Reply:
In the future, we can evaluate if we want to unify even publishing the operator image itself (that can be shared between OLM and remote tests), probably, for this PR, this is enough. wdyt @vmuzikar ?